src/Alley/Validator/BaseValidator.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the laminas-validator-extensions package.
+ *
+ * (c) Alley <info@alley.co>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Alley\Validator;
+
+use Laminas\Validator\AbstractValidator;
+
+abstract class BaseValidator extends AbstractValidator
+{
+    final public function isValid($value)
```

Comment:
Possible to have a return type here, or does that break the function signature extended from the other class?

Reply:
Nope, that can be added.